src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      "
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <!-- <h1>Product cards</h1> -->
+    <div class="card" data-qa="card">
+      <img
+        class="card__logo"
+        src="images/imac.jpeg"
+        alt="Apple iMac"
+      >
+
+      <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+
+      <p class="card__product-code">
+        Код товара: 195434
+      </p>
+
+      <div class="reviews">
+        <div class="reviews__stars">
+          <div class="reviews__star"></div>
+          <div class="reviews__star"></div>
+          <div class="reviews__star"></div>
+          <div class="reviews__star"></div>
+          <div class="reviews__star"></div>
+        </div>
+
+        <p class="reviews__count">
+          Отзывов: 5
+        </p>
+      </div>
+
+      <p class="card__price">
+        <span class="card__price-title">Цена:</span>
+
+        <span class="card__product-price">69 999 грн</span>
```

Comment:
```suggestion
      <p class="card__price">
        <span class="card__price-title">Цена:</span>

        <span class="card__price-price">69 999 грн</span>
```
не так читабельно , але з точки зору БЕМ більш правильно